usng4j-impl/src/test/java/org/codice/usng4j/impl/CoordinateSystemTranslatorTest.java

```diff
@@ -1232,6 +1245,23 @@ public void testUsngToLatLon() throws ParseException {
     }
   }
 
+  @Test
+  public void testParsingInvalidLatitutedZonesToUsng() throws ParseException {
+    final Set<String> invalidUsngStrings =
+        new HashSet<>(
+            Arrays.asList(
+                "15A", "15B", "15I", "15O", "15Y", "15Z", "15K IC", "15K OC", "15K AI", "15K AO",
+                "15K AW", "15K AX", "15K AY", "15K AZ"));
+    for (final String invalidValue : invalidUsngStrings) {
+      try {
+        UsngCoordinateImpl.parseUsngString("18I");
```

Comment:
:exclamation: This code seems to be written to test all `invalidUsngStrings`, but this call is only ever called with `"18I"`! I believe that `"18I"` is supposed to be replaced with `invalidValue`.

Reply:
Good catch!  Fixed.